### PR TITLE
texttest 4.3.0 (new formula)

### PR DESCRIPTION
**Pull request by dafyddj**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/t/texttest.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  license "LGPL-2.1-or-later"
+
+  depends_on "adwaita-icon-theme"
+  depends_on "gtk+3"
+  depends_on "pygobject3"
+  depends_on "python@3.11"
+
+  resource "psutil" do
+    url "https://files.pythonhosted.org/packages/2d/01/beb7331fc6c8d1c49dd051e3611379bfe379e915c808e1301506027fce9d/psutil-5.9.6.tar.gz"
+    sha256 "e4b92ddcd7dd4cdd3f900180ea1e104932c7bce234fb88976e2a3b296441225a"
```
Comment (chenrui333) [inline Formula/t/texttest.rb:17]: depends_on "python-psutil"

and you need to update `pypi_formula_mappings.json` for it as well, many examples there.

File: Formula/t/texttest.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  desc "Tool for text-based Approval Testing"
+  homepage "https://www.texttest.org/"
+  url "https://files.pythonhosted.org/packages/38/22/5b8dbde7193989dccdaf2386f36f335fa83966846f0f17673f9da3b4d6cb/TextTest-4.3.0.tar.gz"
+  sha256 "a183885ec7cb2b6389eb98bf80e84571d93cfa3aeb5e043a6a8c0d0738e39cb4"
+  license "LGPL-2.1-or-later"
+
+  depends_on "adwaita-icon-theme"
+  depends_on "gtk+3"
+  depends_on "pygobject3"
+  depends_on "python@3.11"
```
Comment (chenrui333) [inline Formula/t/texttest.rb:13]: why not py3.12?
```suggestion
  depends_on "python@3.12"
```

File: Formula/t/texttest.rb (hunk 1 above)
Comment (dafyddj) [reply Formula/t/texttest.rb]: Thanks for your help

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @dafyddj for your contribution to homebrew-core!

**Pull request merged**